routes/user.go

```diff
@@ -1078,6 +1104,18 @@ func (fes *APIServer) GetSingleProfile(ww http.ResponseWriter, req *http.Request
 		res.IsGraylisted = true
 	}
 
+	if isAdmin, _ := fes.UserAdminStatus(publicKeyBase58Check); isAdmin {
```

Comment:
Why are we checking if the profile we're fetching is an admin? This seems wrong?

Reply:
Basically, we only need the IsFeaturedTutorialUpAndComing and IsFeaturedTutorialWellKnownCreator values if the user is an admin. These values are only used to determine whether to show "Add" or "Remove" from the well-known and up-and-coming creator indexes on a creator's profile. We don't need to fetch user metadata from global state for anything else, so if we're not an admin viewing a profile, it's a little bit faster.

Reply:
oh I see - yes we need to update this to take in the reader public key. Good catch!